eventually/customprofile/models.py

```diff
@@ -86,7 +87,7 @@ def create(user, hobby='', photo='', birthday=None):
             profile.save()
             return profile
         except (ValueError, IntegrityError):
-            pass
+            LOGGER.error('Inappropriate value or relational integrity fail')
```

Comment:
please use the log messages like convention style on the dev =)

Reply:
done